README.md

```diff
@@ -1,17 +1,228 @@
+  * [How does API support for Kubernetes work?](#how-does-api-support-for-kubernetes-work)
+* [References](#references)
+* [Pre-Requisites](#pre-requisites)
+* [Installing](#installing)
+* [Quick Examples](#quick-examples)
+  * [Deploying a YAML Manifest](#deploying-a-yaml-manifest)
+  * [Deploying a Helm Chart](#deploying-a-helm-chart)
+  * [Deploying a Workload using the Resource API](#deploying-a-workload-using-the-resource-api)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
 
-## Kubernetes version support
+## Introduction
 
-The `pulumi-kubernetes` package closely tracks the latest upstream release, and provides access
-to the full API surface, including deprecated endpoints. At a minimum, we support the current version
-and the previous two versions. Older versions will likely work as well, but are not officially supported.
+`pulumi-kubernetes` provides an SDK to create any of the API resources
+available in Kubernetes.
+
+This includes the resources you know and love, such as:
+- Deployments
+- ReplicaSets
+- ConfigMaps
+- Secrets
+- Jobs etc.
+
+#### Kubernetes API Version Support
+
+The `pulumi-kubernetes` SDK closely tracks the latest upstream release, and provides access
+to the full API surface, including deprecated endpoints.
+The SDK API is 100% compatible with the Kubernetes API, and is
+schematically identical to what Kubernetes users expect.
+
+At a minimum, we support the [current Kubernetes version](https://kubernetes.io/docs/setup/release/) and the previous two versions.
+Older versions will likely work as well, but are not officially supported.
 
 See the [CHANGELOG](https://github.com/pulumi/pulumi-kubernetes/blob/master/CHANGELOG.md) for details
 on supported versions of Kubernetes for each version of the `pulumi-kubernetes` package.
+
+#### How does API support for Kubernetes work?
+
+Pulumi’s Kubernetes SDK is manufactured by automatically wrapping our
+library functionality around the Kubernetes resource [OpenAPI
+spec](https://github.com/kubernetes/kubernetes/tree/master/api/openapi-spec) as soon as a
+new version is released! Ultimately, this means that Pulumi users do not have
+to learn a new Kubernetes API model, nor wait long to work with the latest
+versions available. 
+
+> Note: Pulumi also supports alpha and beta APIs.
+
+Visit the [FAQ](https://www.pulumi.com/docs/reference/clouds/kubernetes/faq/)
+for more details.
+
+## References
+
+* [Reference Documentation](https://www.pulumi.com/docs/reference/clouds/kubernetes/)
+* API Documentation
+    * [Node.js API](https://pulumi.io/reference/pkg/nodejs/@pulumi/kubernetes)
+    * [Python API](https://www.pulumi.com/docs/reference/pkg/python/pulumi_kubernetes/)
+* [All Examples](./examples)
+* [Tutorials](https://www.pulumi.com/docs/reference/tutorials/kubernetes/)
+
+## Pre-Requisites
+
+See the [quickstart](https://www.pulumi.com/docs/quickstart/kubernetes/) for complete documentation.
```

Comment:
This will change to `https://www.pulumi.com/docs/get-started/kubernetes/` once https://github.com/pulumi/docs/pull/1563 gets merged.

Reply:
Thanks, I'll update it.

Are we setting up 301 redirects / alias'ing for it? Historically speaking, I feel like that URL is used in many other places.

Reply:
We are setting up 301 redirects as per https://github.com/pulumi/docs/pull/1606. :)

Reply:
Sorry, I spoke too soon. It looks like @justinvp just ended up adding aliases: https://github.com/pulumi/docs/pull/1563#issuecomment-524388138

@justinvp -- this applies to all pages that have been moved, right?

Reply:
We're using [Hugo `aliases`](https://gohugo.io/content-management/urls/#aliases) for all pages we've moved around within the site. When an alias is used, Hugo generates a page for it with a [meta refresh](https://en.wikipedia.org/wiki/Meta_refresh) -based redirect.

However, with https://github.com/pulumi/docs/pull/1606, any page that has a meta refresh redirect will now be converted into a 301 redirect as part of the Pulumi program that deploys everything to S3.